renv.lock

```diff
@@ -2458,10 +2450,10 @@
     },
     "xfun": {
       "Package": "xfun",
-      "Version": "0.36",
+      "Version": "0.37",
```

Comment:
FYI - I found that updating this to `xfun v0.37` produces an error/warning when downloading reports.

Reply:
Huh, thanks for letting me know about xfun